config/proxy/nginx.conf

```diff
@@ -31,7 +31,7 @@ server {
   }
 
   location / {
-    proxy_pass http://ember:80;
+    proxy_pass http://ember:4200;
```

Comment:
Are you sure?

Reply:
You're right - this would have fixed the local container-based deployment, but broken our actual helm deployment. I made the change in the compose file instead.